README.md

```diff
@@ -100,17 +100,11 @@ npm run test-live
 ## Run end-to-end tests
 In order to run e2e tests you need to install [lisk-core](https://github.com/LiskHQ/lisk)
 
-#### Setup
+#### Setup core
 
-Setup protractor
-
-```
-./node_modules/protractor/bin/webdriver-manager update
-```
-
-Setup a lisk test node to run on localhost:4000 as described in https://github.com/LiskHQ/lisk#tests
-
-And run it with [pm2](http://pm2.keymetrics.io/).
```

Comment:
This line is still needed as `e2e-test-setup.sh` uses `pm2`

Reply:
it automatically starts with that script

Reply:
not if you don't have pm2 installed. So maybe the wording should be "Install [pm2](http://pm2.keymetrics.io/)"